app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
```

Comment:
This and `stream.rb` are almost the same as the `hawkular/middleware` counterparts. Any chance of extracting them to mixins local to the hawkular manager and reuse them?